src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -78,14 +101,22 @@
 
 <script>
 import gql from 'graphql-tag';
-
+import _invokeMap from 'lodash/invokeMap';
+import _mapValues from 'lodash/mapValues';
+import _merge from 'lodash/merge';
+import basicLoanQuery from '@/graphql/query/basicLoanData.graphql';
 import { processPageContentFlat } from '@/util/contentfulUtils';
-import { validateQueryParams } from '@/util/campaignUtils';
+import { validateQueryParams, getPromoFromBasket } from '@/util/campaignUtils';
 import { lightHeader, lightFooter } from '@/util/siteThemes';
+import cookieStore from '@/util/cookieStore';
+// import paginationMixin from '@/plugins/pagination-mixin';
```

Comment:
Started work on both a mixin and util file and will work on moving as much pagination related code as possibly into them a little later.